src/drivers/gitlab.js

```diff
@@ -214,6 +214,8 @@ class Gitlab {
 
       return spawn(command, { shell: true });
     } catch (err) {
+      if (err.message === 'Forbidden')
+        err.message += ', check the permissions of your GitLab token';
```

Comment:
```suggestion
        err.message += ', check the permissions (scopes) of your GitLab token.\nSee https://cml.dev/doc/self-hosted-runners?tab=GitLab#personal-access-token';
```

Reply:
If we had shorter and more stable error links (e.g. https://error.dvc.org/no-dvc-cache as per #606) it would be better, but yes.